right/src/keyboard_layout.h

```diff
@@ -33,6 +33,32 @@ typedef enum {
     UHK_KEY_LPRESSLAYER,
 } uhk_key_type_t;
 
+enum {
```

Comment:
Lets leave the buttons as-is then. Less work for me, and doesn't make a whole lot of difference. The agent doesn't need to expose button composability, so.. I guess all bases covered?